vuln/npm/487.json

```diff
@@ -0,0 +1,22 @@
+{
+  "id": 487,
+  "title": "Arbitrary File Write Through Archive Extraction",
+  "overview": "attackers can write arbitrary files when a malicious archive is extracted.",
+  "created_at": "2019-01-24",
```

Comment:
@sheerun ideally this date will be for when the vulnerability was disclosed to you